src/main/java/mate/academy/spring/validation/RepeatPassword.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.spring.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = RepeatPasswordValidator.class)
+public @interface RepeatPassword {
```

Comment:
you don't use this annotation in your code, so remove it or use it where it's appropriate